site/docs/components/radio-button/usage.mdx

```diff
@@ -20,6 +20,7 @@ data:
 - When you have fewer than five options and multiple selection is necessary. Instead, use [`CheckboxGroup`](../checkbox).
 - When you have more than five and fewer than 10 options. Instead, use [`Dropdown`](../dropdown).
 - When you have more than 10 options. Instead, use [`ComboBox`](../combo-box).
+- When it should be possible to deselect after a single option has been selected. Instead, consider [`Switch`](../switch), [`Checkbox`](../checkbox) or [`Dropdown`](../dropdown).
```

Comment:
Has this been checked by design? Have we thought about encouraging the usage of a reset button instead?